README
Useful informations in the reports module : xclass, ajax, cliKeys, eID, general status of the system (encoding, DB, php vars...), hooks, compare local and TER extension (diff), used content type, used plugins, ExtDirect... It can really help you during migration or new existing project (to have a global reports of the system).

TER url : http://typo3.org/extensions/repository/view/additional_reports

ChangeLog
2012-05-04 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Fix bug #36788
	* Fix bug #36786
2012-04-27 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Big refactoring/cleanup using a backend template rendering
2012-03-27 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* fix tstamp in errors log
2012-03-06 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Fix isLoaded() for realurl in autoload.php
2012-03-03 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Security fix, please update
2012-02-20 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Optimizations
2012-02-20 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Bugfix on eid list
2012-02-13 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Add areports to check database integrity (like install tool)
	* Another refactoring
	* Small features on a lot of reports :-)
2012-01-02 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Fix deprecated functions of the install tool
2011-12-24 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Bugfix on conf update check
2011-11-24 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* In status report : add some features like apache user/group, status of the mysql cache, db size...
2011-11-09 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* new feature : check if an extension need ext conf update (usefull after mass update)
2011-10-27 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Add the number of pages for each domain
2011-10-14 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Update the mod1 for 4.3< versions
	* Add a query to flush errors log
2011-10-07 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Fix for TYPO3 4.6 compatibility
2011-10-04 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Add a new report : check your websites/domains/baseURL configuration
	* Fix the pagebrowser in errors log
2011-08-30 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* In plugin list view, you can now switch into the list or page module
	* In "List of used content types" & "List of used plugins", do not display offline pages
	* In "List of used content types" & "List of used plugins", add first domain record
2011-08-16 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Small bugfix in the ExtDirect reports
2011-08-15 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Add the PHP errors log
	* Change the extensions reports
	* In the extensions reports, add the possibility to check in the emconf if the extension is declared compatible with the current TYPO3 version
2011-07-27 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Add realurl errors log
2011-06-17 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Do not display standard page link with TemplaVoilà
2011-05-25 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Small bugfixes
	* Complete refactoring
	* Add a support for version lower than 4.3 (with a module instead of reports)
2011-05-10 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Now compare the local version of a file to the same TER version (before it was the last version)
	* Ignore blank spaces in diff (-b option)
2011-05-08 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Update some labels
	* Bug on a label in plugins mode
 	* Add a german version (thanks to Stefan Galinski)
2011-05-06 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* In extension list mode, add a shadowbox to the db part
	* Add a ExtDirect list module (in 4.5>= only)
2011-04-13 CERDAN Yohann <cerdanyohann@yahoo.fr>
    * In plugins and contents list, add the possibility to show hidden pages/contents
2011-04-11 CERDAN Yohann <cerdanyohann@yahoo.fr>
    * Nice feature : in extension list mode you can view the modified files in linux like diff mode (local file/TER file)
2011-03-31 CERDAN Yohann <cerdanyohann@yahoo.fr>
    * Add a new module to list extension with : TER update needed? database update needed? modified files? etc...
2011-03-08 CERDAN Yohann <cerdanyohann@yahoo.fr>
    * Small bugfix for view icon on TYPO3 4.3
    * Add a filter to filter by content in the "List of used content" view
    * Bugfix : EM link was broken
2011-02-28 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Small bugfix for icons (not displayed if you have your TYPO3 installation in a subfolder).
2011-02-11 CERDAN Yohann <cerdanyohann@yahoo.fr>
    * Add a new view to display the available BE ajax methods
2011-02-09 CERDAN Yohann <cerdanyohann@yahoo.fr>
    * Add a filter to filter by plugins in the "List of used plugins" view
2011-02-05 CERDAN Yohann <cerdanyohann@yahoo.fr>
    * Add a typo3 page brower on the "List of used content types & "List of used plugins" view
    * Check if the content/plugin is used in TV (non used elements)
    * Use the jumpurl method instead of teh submit button
    * Save the last menu you use (in the plugins section)
2011-01-03 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Add a link to the page module & page TV module (if TV is loaded) in the table
2010-12-27 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Bugfix : can't get the mysql informations in some server
	* Bugfix : correct link to the EM
2010-12-23 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Bugfix : test if the apache_get_version function exist
2010-10-10 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Add hook support (need feedbacks about it)
	* Add a status module (with informations about typo3,php,apache,mysql,crontab...)
	* Update the locallang
	* Refactoring HTML & PHP code
	* Update the manual with new screenshots
2010-10-05 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Update the locallang
	* Update the manual
	* Add a new option 'Summary' in the 'plugins & content type' module
2010-10-03 CERDAN Yohann <cerdanyohann@yahoo.fr>
	* Initial release



ext_emconf.php
<?php

/***************************************************************
 * Extension Manager/Repository config file for ext "additional_reports".
 *
 * Auto generated 04-12-2015 14:39
 *
 * Manual updates:
 * Only the data in the array - everything else is removed by next
 * writing. "version" and "dependencies" must not be touched!
 ***************************************************************/

$EM_CONF[$_EXTKEY] = array (
	'title' => 'Useful informations in reports module',
	'description' => 'Useful informations in the reports module : xclass, ajax, cliKeys, eID, general status of the system (encoding, DB, php vars...), hooks, compare local and TER extension (diff), used content type, used plugins, ExtDirect... It can really help you during migration or new existing project (to have a global reports of the system).',
	'category' => 'misc',
	'shy' => 0,
	'version' => '2.5.10',
	'dependencies' => '',
	'conflicts' => '',
	'priority' => '',
	'loadOrder' => '',
	'TYPO3_version' => '4.0.0-6.0.0',
	'PHP_version' => '5.0.0-0.0.0',
	'module' => '',
	'state' => 'stable',
	'uploadfolder' => 0,
	'createDirs' => '',
	'modify_tables' => '',
	'clearcacheonload' => 0,
	'lockType' => '',
	'author' => 'CERDAN Yohann',
	'author_email' => 'cerdanyohann@yahoo.fr',
	'author_company' => '',
	'CGLcompliance' => NULL,
	'CGLcompliance_note' => NULL,
);

?>